Command/DoctrineCommand.php

```diff
@@ -25,11 +25,11 @@
  */
 abstract class DoctrineCommand extends BaseCommand
 {
-    public static function configureMigrations(ContainerInterface $container, Configuration $configuration) : void
+    public static function configureMigrations(ContainerInterface $container, Configuration $configuration): void
     {
         $dir = $configuration->getMigrationsDirectory();
 
-        if (empty($dir)) {
+        if ($dir === null) {
```

Comment:
`$dir` could be an empty string in theory, right?

Reply:
Correct, but did we really mean to catch this case here?

Reply:
No, definitely not here, but I can't evaluate the impact of this BC break. It looks like it won't have much impact.

Reply:
Yeah I can't say I can evaluate it. I guess you're right, I should use a more complex condition just to be sure, the thing is, I think the original author would have picked `$dir === null` if `empty` had been forbidden earlier. It's what I believe but it's just that: a belief :sweat_smile:

Reply:
Let's go with our gut.